patches/minecraft/net/minecraft/client/audio/SoundHandler.java.patch

```diff
@@ -0,0 +1,33 @@
+--- ../src-base/minecraft/net/minecraft/client/audio/SoundHandler.java
++++ ../src-work/minecraft/net/minecraft/client/audio/SoundHandler.java
+@@ -67,6 +67,7 @@
+     {
+         this.field_147697_e.func_148763_c();
+ 
++        Map<ResourceLocation, SoundList> resources = new java.util.IdentityHashMap<>();
+         for (String s : p_110549_1_.func_135055_a())
+         {
+             try
```

Comment:
How about this try-catch phrase? I fear that unexpected consequences may emerge once this statement is removed (as in the consumer below)

Reply:
Resolved.